base/lthooks.dtx

```diff
@@ -340,6 +346,16 @@
 %
 % \subsubsection{Updating code for hooks}
 %
+% In contrast to the commands from the previous section, declarations
+% such as \cs{AddToHook} or \cs{DeclareHookRule} are allowed to be
+% used even if the hook is not yet declared. The rationale is, that
+% the hook declaration may be in some package that is loaded later or
+% perhaps not loaded at all.
+%
+% A side effect of this design is that misspelling do not raise an
```

Comment:
```suggestion
% A side effect of this design is that misspellings do not raise an
```

Either "misspelling does not" or "misspellings do not" is ok. Use the second one to be consistent with that in the doc for hook rules.